config/routes.rb

```diff
@@ -1,3 +1,5 @@
+require 'sidekiq/web'
+
```

Comment:
I think that misc views with api isn't good. We cannot monitor sidekiq from cli ?